src/quantum/setup.py

```diff
@@ -17,7 +17,7 @@
 # This version should match the latest entry in HISTORY.rst
 # Also, when updating this, please review the version used by the extension to
 # submit requests, which can be found at './azext_quantum/__init__.py'
-VERSION = '0.21.0'
+VERSION = '0.22.0'
```

Comment:
Now we support new versioning for extension and 0.x.x is not recommended any longer. The version will start from 1.0.0 (stable) or 1.0.0b1 (preview). You can find more details in  https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md.
Please update the version to correct one.

Reply:
Even with preview service?

Reply:
Yes, please don't use `0.x.x` any longer. If your service is in preview, please use `1.0.0b1` instead.

Reply:
updated. Could you please take another look?